### PR TITLE
Added  sample_rate for the sensors

### PR DESCRIPTION
**Pull request by SenexCrenshaw**
## Description:
Updated to match the below PR

**Pull request in [esphome](https://github.com/esphome/esphome) with YAML changes (if applicable):** esphome/esphome#1688

## Checklist:

  - [X] Branch: `next` is for changes and new documentation that will go public with the next ESPHome release. Fixes, changes and adjustments for the current release should be created against `current`.
  - [ ] Link added in `/index.rst` when creating new documents for new components or cookbook.

### CODE REVIEW

File: components/sensor/bme680_bsec.rst [hunk 1]
```diff
@@ -174,59 +178,100 @@ For each sensor all other options from :ref:`Sensor <config-sensor>` and :ref:`T
...
-
         # Interval at which to save BSEC state
         # ------------------------------------
         # Default: 6h
         state_save_interval: 6h
 
     sensor:
       - platform: bme680_bsec
-        temperature:
+        temperature:
```
Comment (trvrnrth) [inline components/sensor/bme680_bsec.rst]: I can't see anything else wrong so I'll just point out these couple of bits of trailing whitespace!

**Review by trvrnrth — APPROVED** [review]

**Comment by trvrnrth** [comment]
Superseded by #1108

**Review by trvrnrth — COMMENTED** [review]
Superseded by #1108

**Comment by SenexCrenshaw** [comment]
Closed Superseded by #1108

**Pull request closed without merging**